react/features/large-video/components/styles.js

```diff
@@ -32,6 +27,17 @@ export default createStyleSheet({
         right: FILMSTRIP_SIZE
     },
 
+    labelWrapper: {
```

Comment:
I don't think this should be here. In fact, I don't think the labels should be part of LargeVideo at all. Can we move them to Conference, which already takes care of this?

Reply:
You may be right, but this change would also involve web, and labels on web are handled differently. I remember I originally followed the web path when implemented the labels (somewhere around summer) and it stayed here. I'm happy to spend some more time on it to move it to Conference (including web, of course), but I hope this is not the scope of this PR now